README.md

```diff
@@ -213,7 +213,7 @@ For security issues, kindly email us at [security@appwrite.io](mailto:security@a
 
 ## Follow Us
 
-Join our growing community around the world! Check out our official [Blog](https://medium.com/appwrite-io). Follow us on [Twitter](https://twitter.com/appwrite), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/), [Dev Community](https://dev.to/appwrite) or join our live [Discord server](https://discord.gg/GSeTUeA) for more help, ideas, and discussions.
+Join our growing community around the world! Check out our official [Blog](https://medium.com/appwrite-io). Follow us on [Twitter](https://twitter.com/appwrite), [Facebook Page](https://www.facebook.com/appwrite.io), [Facebook Group](https://www.facebook.com/groups/appwrite.developers/), [Dev Community](https://dev.to/appwrite),[LinkedIn](https://www.linkedin.com/company/appwrite/?originalSubdomain=il) or join our live [Discord server](https://discord.gg/GSeTUeA) for more help, ideas, and discussions.
```

Comment:
Change linkedin link:- https://www.linkedin.com/company/appwrite/

Reply:
> Change linkedin link:- https://www.linkedin.com/company/appwrite/

Changed the link! Ok, I'll remove the logo file